.readthedocs.yaml

```diff
@@ -0,0 +1,14 @@
+# .readthedocs.yml
+version: 2
+
+build:
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.12"
+
+python:
+  install:
+    - requirements: docs/requirements-docs.txt
+
+mkdocs:
+  configuration: mkdocs.yml
```

Comment:
Is the configuration ```mkdocs.yml``` supposed to be added later?

Reply:
Done